src/module.ts

```diff
@@ -100,7 +104,10 @@ const defaults: ModuleOptions = {
     idLength: 64,
     storePrefix: 'sessions',
     cookieSameSite: 'lax',
-    storageOptions: {}
+    storageOptions: {
+      driver: 'memory',
+      options: {}
+    }
```

Comment:
Please also update this in the main readme `default module configuration` section!

I would also add an addition below section about possibility to use different drivers with at least one other driver example 🙂

Reply:
Updated the readme, also added an example for Redis.